packages/lib-classifier/src/translations/README.md

```diff
@@ -32,7 +32,7 @@ const MyComponent = () => {
 
 FEM is integrated with [Lokalise](https://app.lokalise.com) for translations management. Any dictionary other than `/en` should be managed through the Lokalise Dashboard. Instructions on how to import and export dictionary files can be found ________ (A link will be added here when a Lokalise playbook is published on Zooniverse's Github).
 
-New languages files must be added in the `supportedLngs` array in [i18n.js](i18n.js).
+Check that added language is in the `supportedLngs` array in [i18n.js](i18n.js).
```

Comment:
Should that file be edited in this PR, or is it waiting for new languages to be added by translators?